pkg/provision/buildroot.go

```diff
@@ -170,3 +177,90 @@ func setRemoteAuthOptions(p provision.Provisioner) auth.Options {
 
 	return authOptions
 }
+
+func configureAuth(p *BuildrootProvisioner) error {
+	driver := p.GetDriver()
+	machineName := driver.GetMachineName()
+	authOptions := p.GetAuthOptions()
+	org := mcnutils.GetUsername() + "." + machineName
+	bits := 2048
+
+	ip, err := driver.GetIP()
+	if err != nil {
+		return errors.Wrap(err, "error getting ip during provisioning")
+	}
+
+	if err := util.CopyFile(authOptions.CaCertPath, filepath.Join(authOptions.StorePath, "ca.pem")); err != nil {
```

Comment:
Oops, I think that was the point of the map down there anyways.  I'll change that